project/threads/templates/threads/static_templates/landing.html

```diff
@@ -57,32 +55,32 @@ <h3 class="content-1-title bold">CiviWiki is a non-profit community working to d
         </div>
         <div class="video full-height valign-wrapper">
             <div class="container section valign">
-                <h4 class="section center white-text">Check out our short video describing CiviWiki...</h4>
+                <h4 class="section center white-text">{% trans 'Check out our short video describing CiviWiki' %} ...</h4>
                 <p class="center"></p>
                 <div class="videoWrapper">
                     <iframe width="560" height="315" src="https://www.youtube.com/embed/Ll4VZv4a8bE" frameborder="0" allowfullscreen></iframe>
                 </div>
-                <div class="grey-text text-darken-3 center">Video made by <a class="grey-text text-darken-1" href="https://sheldonanim.com/" target="_blank">Jon Sheldon</a></div>
+                <div class="grey-text text-darken-3 center">{% trans 'Video made by' %}  <a class="grey-text text-darken-1" href="https://sheldonanim.com/" target="_blank">Jon Sheldon</a></div>
             </div>
         </div>
 
         <div class="why-civiwiki container cushion">
             <div class="section">
-                <h2 class="bold center content-title">Why CiviWiki?</h2>
+                <h2 class="bold center content-title">{% trans 'Why' %} CiviWiki?</h2>
```

Comment:
The full sentence is important to the translator.

```suggestion
                <h2 class="bold center content-title">{% trans 'Why CiviWiki?' %}</h2>
```